orttraining/orttraining/training_ops/rocm/tensor/gather_grad_impl.cu

```diff
@@ -150,39 +102,28 @@ void GatherGradImpl(
       nullptr, sort_temp_storage_size_bytes,
       indices_data, indices_data_sorted.get(),
       original_indices.get(), original_indices_sorted.get(),
-      num_indices, 0, sizeof(Tin)*8, stream));
+      num_indices));
 
   auto sort_temp_storage = rocm_kernel.GetScratchBuffer<void>(sort_temp_storage_size_bytes);
 
   HIP_CALL_THROW(hipcub::DeviceRadixSort::SortPairs(
       sort_temp_storage.get(), sort_temp_storage_size_bytes,
       indices_data, indices_data_sorted.get(),
       original_indices.get(), original_indices_sorted.get(),
-      num_indices, 0, sizeof(Tin)*8, stream));
+      num_indices));
 
   dim3 block(GPU_WARP_SIZE, 4);
-  dim3 grid(CeilDiv(num_indices, 4), CeilDiv(stride, GridDim::maxElementsPerThread * GPU_WARP_SIZE));
-  if (param_itrs == 1)
```

Comment:
Instead of reverting the change, I would prefer to commenting or removing the code #165-#175 as below and keeping the optimized kernel code.

Reply:
Yeah, makes sense. Let me make validate that change and update PR.